src/test/java/tictactoe/PartyCreatorTest.java

```diff
@@ -0,0 +1,14 @@
+package tictactoe;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PartyCreatorTest {
+    @Test
+    public void itCreatesAParty() {
+        PartyCreator partyCreator = new PartyCreator();
+        Party party = partyCreator.newParty(null, 3, GameTypes.COMPUTER_VS_COMPUTER);
```

Comment:
When we pass null around it doesn't give us the context. I realise that the `io` is not important for this test, however you could assign a variable like `IO unusedIo = null;` then pass in `newParty(unusedIo, 3, ...);` so that it's clear it's unused but also clear what it should be in real life.